### ISSUE TITLE
helpers.Page.pager cannot be configured to add accessibility spans

### ISSUE THREAD

**Issue by J-bytes** [ckan/ckan#5733]
**CKAN version=2.8.6**

**Describe the bug**
Site should be WCAG accessible

**Steps to reproduce**
Cannot modify style of the helpers.Page.pager so that proper accessibility spans can be introduced

**Expected behavior**
The helpers.Page.pager shoud be replaced by a snippet to allow modification and the introduction of <span> to allow WCAG compliance

**Additional details**
Duplicate of https://github.com/ckan/ckan/issues/3104 which pointed out the same problem in ckan 2.8.6

**Comment by amercader** [comment]
@smotornyuk worked on a pagination refactor for 2.9 which I believe fixed this. @smotornyuk if that is the case can you point to the commits / PR for reference?

**Comment by smotornyuk** [comment]
I've [changed a code](https://github.com/ckan/ckan/pull/5223/files#diff-31de01cd67935c2e74f699956fe34991dd26f8b9ba284865955cf6f35734c5d0L629-L634) so that labels of the **Next/Prev** page, class of the current page, and page-link attributes can be controlled. As I see people want is a way to rewrite the pagination template,  not messing with the `helpers.Page` configuration.

Looking at the current implementation, I think, that we can provide a few very simple templates: **pagination-wrapper**, **page-link**, **dot-dot-link**(a segment that is shown when there are too many pages). 

If this issue can wait till January, you can assign it to me. Right now I'm a bit overwhelmed with internal work, but I'll have few lazy weeks in January:)
